package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "tsc",
     "prepublishOnly": "npm run build",
-    "test": "mocha",
+    "test": "node --no-warnings --loader tsx --test test/api.spec.ts",

```

Comment:
Should be enough here:

`node --loader tsx --test test/`

Reply:
cant seem to work tho it dosent detect tests and display there are 0 tests